tickit/adapters/zmqadapter.py

```diff
@@ -0,0 +1,113 @@
+import asyncio
+import logging
+from dataclasses import dataclass
+from typing import AsyncIterable, Awaitable, Callable, List
+
+import aiozmq
+import zmq
+
+from tickit.core.adapter import ConfigurableAdapter
+from tickit.core.device import ConfigurableDevice
+
+LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class ZeroMQStream(ConfigurableDevice):
```

Comment:
Since that has no zmq-specific logic in it I wouldn't give a zmq-specific name. Something like `CountingDevice` would be fine

Reply:
Made in #29